conf/ledgersmb-apache-2.4.template

```diff
@@ -0,0 +1,60 @@
+# This file must be edited to substitute the current path with WORKING_DIR
+
+Alias /lsmbcss /var/lib/master/css/
+
+<VirtualHost *:80>
+
+  ServerName [SERVERNAME]:80
+  DocumentRoot WORKING_DIR
+
+  <Proxy *>
+    Order Allow,Deny
+    Allow from All
+  </Proxy>
+
+  LogLevel Warn
+
+  <FilesMatch ...>
+    SetHandler  "proxy:pl://localhost:5001"
+  </FilesMatch>
+
+  ProxyRequests     Off
+  ProxyPreserveHost On
+  ProxyVia          On
+  ProxyPass        / http://localhost:5001/
+  ProxyPassReverse / http://localhost:5001/
+
+</VirtualHost>
+
+# The rest of this file just tightens up security.
+<LocationMatch "/ledgersmb/.+\.conf$">
+  Require all denied
+</LocationMatch>
+
+<LocationMatch /ledgersmb/bin>
+  Require all denied
+</LocationMatch>
+
+<LocationMatch /ledgersmb/utils>
+  Require all denied
+</LocationMatch>
+
+<LocationMatch /ledgersmb/spool>
+  Require all denied
+</LocationMatch>
+
+<Directory /var/share/master/templates>
+  Require all denied
+</Directory>
+
+<LocationMatch /ledgersmb/master>
+  Require all denied
+</LocationMatch>
+
+<LocationMatch /lsmbcss>
```

Comment:
This line is shadowed by the alias above. Ideally, we want to put the alias and any access restrictions in the URL space (which the aliasmatch would be), into the virtualhost block, so one URL isn't accessible through multiple vhosts.